admin/src/utils/wrapWithErrorHandling.js

```diff
@@ -0,0 +1,3 @@
+export default cb => (effects, ...args) => {
+    return cb(effects, ...args).then(value => value, error => effects.setError(error.message));
```

Comment:
Add unit test, and what if cb does not return a promise, or throw an error directly ?